library/src/main/resources/base/element/string.json

```diff
@@ -0,0 +1,8 @@
+{
+  "string": [
+    {
+      "name": "library_library",
+      "value": "library_library"
```

Comment:
The string value is very generic. Can be more meaningful & specific.

Reply:
Done